### DISCUSSION TITLE
How to integrate with goland code inspection?

### DISCUSSION THREAD

**Discussion by Burning1020** [golangci/golangci-lint#5185] [Q&A] [answered]
I have installed the golangci-lint plugin on Goland, configured to file watcher, and everything is ok!

However, I also expect linter result can be showed in real-time, not just by file saved. That is pretty look-like code inspections, it gives you errors/warning/typo hint on the top-right window or show wavy lines under code, so I wander if our plugin can give the same behavior?

**Comment by ldez** [comment, marked as answer]
FYI, we are not the maintainers of the [Goland plugin for golangci-lint](https://plugins.jetbrains.com/plugin/12496-go-linter), this plugin is developed by JetBrains.
But I guess it just runs the binary.

So golangci-lint needs files to analyze, your files need to be on your file system.

There is no "real-time" inspection if the changes are not saved.

The plugin is not a code inspection: code inspections are rules inside the IDE.
As you said, the plugin is run by a file watcher, so when files are saved.

**Reply by Burning1020** [reply]
OK.Thank you.
